src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -190,5 +240,25 @@ public int Height
         ///    </para>
         /// </summary>
         public override string ToString() => "{Width=" + _width.ToString() + ", Height=" + _height.ToString() + "}";
```

Comment:
Didn't realize it was existing, looked green in the diff. That's a better question for @mellinoe then.